database/storeprocds/spInsertTransactionType.sql

```diff
@@ -0,0 +1,31 @@
+-- =============================================
+-- Create basic stored procedure template
+-- =============================================
+
+-- Drop stored procedure if it already exists
+IF EXISTS (
+  SELECT * 
+    FROM INFORMATION_SCHEMA.ROUTINES 
+   WHERE SPECIFIC_SCHEMA = N'dbo'
+     AND SPECIFIC_NAME = N'spInsertTransactionType' 
+)
+   DROP PROCEDURE dbo.spInsertTransactionType
+GO
+
+CREATE PROCEDURE dbo.spInsertTransactionType
+	@Name nvarchar(10),
+	@Description nvarchar(50)
+AS
+	IF (@Name IS NOT NULL)
+	BEGIN
+	INSERT INTO TRANSTYPE(Name,Description)
```

Comment:
if table field is same as sys reserved keyword so write like this [Name], [Description]

Reply:
fixed